Library/Navigation.swift

```diff
@@ -221,6 +224,7 @@ extension Navigation {
 
 private let routes: [String:(RouteParams) -> Decoded<Navigation>] = [
   "/": emailClick,
+  "/mpss/:a/:b/:c/:d/:e/:f/:g": emailLink,
```

Comment:
would be nice  if we could match against urls with wildcards, like `/mpss/*`, but i don't have time to flesh that out and it seems all email urls are of this form with 7 path components after `mpss`